modules/cluster-logging-deploy-console.adoc

```diff
@@ -39,162 +40,14 @@ include::snippets/logging-stable-updates-snip.adoc[]
 .Verification
 
 . Verify that the *Red Hat OpenShift Logging Operator* is installed by switching to the *Operators* → *Installed Operators* page.
+. In the *Status* column, verify that you see green checkmarks with *InstallSucceeded* and the text *Up to date*.
```

Comment:
Are *InstallSucceeded* and *Up to date* fields? Maybe the word "field" could be implemented so that both these entries do not seem slightly different.

Reply:
Not sure, since this came from preexisting content